clusters/script-gen-readonly-kubeconfig.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+cat <<EOF | kubectl apply -f -
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: crole-customresources-readyonly
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
+rules:
+- apiGroups:
+    - source.toolkit.fluxcd.io/v1beta2
```

Comment:
`apiGroups` need to be provided without the `version` part of the name, so this needs to be:

```suggestion
    - source.toolkit.fluxcd.io
```